packages/odyssey-storybook/src/components/odyssey-mui/Icon/Icon.stories.tsx

```diff
@@ -160,31 +169,31 @@ const meta: Array<{ name: keyof typeof iconDictionary; use: string }> = [
 ];
 
 export const Library = (): ReactElement => {
+  // return <div />;
   return (
-    <Table caption="Library" screenReaderCaption="Available Odyssey Icons">
-      <Table.Header>
```

Comment:
Why did we remove the caption? I thought the icon stories were using Odyssey-MUI. Is this old Odyssey-React?